[DKP] Release Notes 2.2.2 (#4567)
mesosphere/dcos-docs-site: pages/dkp/konvoy/2.2/release-notes/2.2.2/index.md

### Kiali Platform Application not working properly (COPS-7336)

An installation issue that caused the Kiali platform application to malfunction has been corrected.

### Updated Dex CRDS (COPS-7341)

The DEX Custom Resource Definitions used for configuring LDAP have been updated to include the correct syntax.

### DKP custom credentials plugin restored (COPS-7343)

The mesosphere/dex-k8s-authenticator docker container now includes the appropriate binaries that allow users to download the referenced 'konvoy-async-plugin' after configuring a cluster using an external IDP for authentication.

## Component updates

When upgrading to this release, the following services and service components are upgraded to the listed version:

| Common Application Name | APP ID                                         | Version | Component Versions                                                                                                                        |
| ----------------------- | ---------------------------------------------- | ------- | ----------------------------------------------------------------------------------------------------------------------------------------- |

> | Cert Manager            | cert-manager                                   | 1.7.1   | - chart: 1.7.1<br>- cert-manager: 1.7.1                                                                                                 |

Reviewer: Did these get updates from Brian Mendoza jira?

---
layout: layout.pug
navigationTitle: DKP 2.2.2 Release Notes
title: DKP 2.2.2 Release Notes

> menuWeight: 20

Reviewer 2: So this file appears below 2.2.1 RN
```suggestion
menuWeight: 30
```

<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. New customers must contact their sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download or install DKP.</p>

## Release summary

Welcome to D2iQ Kubernetes Platform (DKP) 2.2.2! This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintains compatibility and support for other packages used in Konvoy.

DKP 2.2.x supports Kubernetes versions between 1.21.0 and 1.22.x. Any cluster you want to attach using DKP 2.2.x must be running a Kubernetes version in this range.

### Supported versions

| Kubernetes Support | Version |
| ------------------ | ------- |
| **Minimum**  | 1.21.0  |
| **Maximum**  | 1.22.x  |
| **Default**  | 1.22.8  |

## Fixes and Improvements

### Fixed containerd vulnerability  (COPS-7531)

DKP packages now include updated containerd and FIPS manifests, mitigating CVE-2022-23648, a known containerd vulnerability.

> ### Dex CRDS are out of date (COPS-7341)

Reviewer 2: ```suggestion
### Outdated Dex CRDS (COPS-7341)
```

The custom resource definitions used for configuring LDAP have been updated to include the correct syntax.

> ### The Konvoy credentials plugin instructions embedded in the dex-k8s-authenticator containers do not work (COPS-7343)

Reviewer 2: ```suggestion
### The Konvoy credentials plugin instructions embedded in the dex-k8s-authenticator containers did not work (COPS-7343)
```

> ### The Konvoy credentials plugin instructions embedded in the dex-k8s-authenticator dialogs don't work (COPS-7343)

Reviewer 2: ```suggestion
### The Konvoy credentials plugin instructions embedded in the dex-k8s-authenticator dialogs do not work (COPS-7343)
```
